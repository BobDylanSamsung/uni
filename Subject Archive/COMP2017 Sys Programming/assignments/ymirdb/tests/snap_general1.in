set a 1 2 3
snapshot
set b 1 2 a
snapshot
checkout 1
list entries
checkout 2
list entries
bye
