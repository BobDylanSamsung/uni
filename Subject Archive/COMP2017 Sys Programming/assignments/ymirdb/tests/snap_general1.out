ok

saved as snapshot 1

ok

saved as snapshot 2

ok

a [1 2 3]

ok

b[1 2 a]
a[1 2 3]

bye
